client/src/components/core/ms-dropdown/index.ts

```diff
@@ -0,0 +1,4 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+import MsDropdown from '@/components/core/ms-dropdown/MsDropdown.vue';
+
+export { MsDropdown };
```

Comment:
Probably needs to export the `MsDropdownChangeEvent` too. It may not be explicitly required but it could be used to type a variable (something like `async def myFunction(event: MsDropdownChangeEvent): Promise<void>`

Reply:
missed it. Done!